api/src/main/java/org/openmrs/module/appointmentscheduling/api/impl/AppointmentServiceImpl.java

```diff
@@ -1253,7 +1253,7 @@ public void purgeProviderSchedule(ProviderSchedule providerSchedule) {
 
 	@Override
 	public List<ProviderSchedule> getProviderSchedulesByConstraints(Location location, Provider provider, List<AppointmentType> appointmentTypes) {
-		return getProviderScheduleDAO().getProviderScheduleByConstraints(location, provider,null);
+		return getProviderScheduleDAO().getProviderScheduleByConstraints(location, appointmentTypes.get(0), provider,null);
```

Comment:
Why does this only pass the first appointment type in the file to the DAO method?   Either the underlying DAO method should support a List<AppointmentType> or the getProviderSchedulesByConstraints should only take in an AppointmentType...

Reply:
@mogoodrich I'm reusing the same method here, https://github.com/openmrs/openmrs-module-appointmentscheduling/blob/6858eafde1bf3c59375acef43ab0e887ef52909c/omod/src/main/java/org/openmrs/module/appointmentscheduling/rest/resource/openmrs1_9/ProviderScheduleResource1_9.java#L129

that's why it's taking list of appointment types as arguments

Reply:
@jecihjoy looks like this part of the change set is outdated anyway?

But this reminds me... are you still awaiting approval of this change set for merging?  I thought we had finished everything and this dropped off my radar... sorry if that was't the case!